core/editor/src/document/document_message_handler.rs

```diff
@@ -95,11 +96,24 @@ impl MessageHandler<DocumentMessage, ()> for DocumentMessageHandler {
 				self.active_document_mut().layer_data(&path).expanded ^= true;
 				responses.extend(self.handle_folder_changed(path));
 			}
+			DeleteSelectedLayers => {
+				// TODO: Replace with drain_filter https://github.com/rust-lang/rust/issues/59618
+				log::debug!("{:?}", self.active_document().layer_data);
+				let paths: Vec<Vec<LayerId>> = self.active_document().layer_data.iter().filter_map(|(path, data)| data.selected.then(|| path.clone())).collect();
```

Comment:
Do we have a type alias for `Vec<LayerId>` or `Vec<Vec<LayerId>>`? If we do have one let's not forget to use it here.

Reply:
We don't